client/js/widgets/widget.js

```diff
@@ -2330,9 +2331,10 @@ export class Widget extends StateManaged {
       if(!o.confirmButtonText && !o.confirmButtonIcon){
         confirmButtonText = "Go";
       }
-      if(!o.cancelButtonText && !o.cancelButtonIcon){
+      if (o.cancelButtonText === null && o.cancelButtonIcon === null) {
+        $('#buttonInputCancel').style.visibility = "hidden";
+      } else (!o.cancelButtonText && !o.cancelButtonIcon)
```

Comment:
This needs to be `else if`.

Reply:
Good catch.  I wish I knew how to code.  Thanks for putting up with me.